docs/root/version_history/current.rst

```diff
@@ -105,6 +105,7 @@ New Features
   to the DNS cache implementation to configure the minimum DNS refresh rate, regardless of returned
   TTL. This was previously hard coded to 5s and defaults to 5s if unset.
 * http: added random_value_specifier in :ref:`weighted_clusters <envoy_v3_api_field_config.route.v3.RouteAction.weighted_clusters>` to allow random value to be specified from configuration proto.
+* http: added request_mirror_policies to higher levels ``RouteConfiguration`` and ``VirtualHost`` which applies to :ref:`request_mirror_policies <envoy_v3_api_field_config.route.v3.RouteAction.request_mirror_policies>` in all routes underneath without configured mirror policies..
```

Comment:
```suggestion
* http: added request_mirror_policies to higher levels ``RouteConfiguration`` and ``VirtualHost`` which applies to :ref:`request_mirror_policies <envoy_v3_api_field_config.route.v3.RouteAction.request_mirror_policies>` in all routes underneath without configured mirror policies.
```

Also, please ref link to both newly added fields.

Reply:
done